docs/contributor/arch/013-resource-attributes-enrichment-for-non-workload-metrics.md

```diff
@@ -63,7 +63,7 @@ Load test using `europe-docker.pkg.dev/kyma-project/dev/kyma-otel-collector:PR-1
   - Clean solution. The non-workload metrics will never have the unwanted resource attributes set to any value.
   - This is the recommended solution for doing a conditional routing in an OTel collector pipeline.
 - _Cons_:
-  - Per MetricPipeline, we will have 3 pipelines in the collector instead of 1. So, we will have a more complex setup of the pipeline definitions in combination with the new connector definitions.
+  - Per MetricPipeline, we will have 3 pipelines in the collector instead of 1. So, we will have a more complex setup of the pipeline definitions in combination with the new connectors definitions.
```

Comment:
```suggestion
  - Per MetricPipeline, we will have 3 pipelines in the collector instead of 1. So, we will have a more complex setup of the pipeline definitions in combination with the new connector's definitions.
```
I'm guessing this is genitive? Plural sounds weird to me

Reply:
we have 2 connectors. So, it should be plural

Reply:
I changed the structure of the sentence. Can you have another look?